### ISSUE TITLE
multilang seed?

### ISSUE THREAD

**Issue by krtschmr** [evanrolfe/monero_wallet_gen#1]
https://github.com/evanrolfe/monero_wallet_gen/blob/ccfcf1fcb900b2cfecb168dd48b162c1c16b6566/lib/monero_wallet_gen/words.rb#L1

    WORDS = [

should be changed into

    WORDS = {en:, de:, es:}

and then either we need set a config parameter `monero_wallet_gen.language = :pt` or if we create a wallet we need to give in the `lang` which default is `en`

**Comment by evanrolfe** [comment]
Good idea, I lean towards having `lang` as a parameter to the generate method just like `testnet` just to keep it simple. i.e.
```ruby
MoneroWalletGen::generate(lang: :de)
```

This shouldn't be too hard to do and would be a welcome contribution if somebody wants to submit a pull request for this change!
